src/Reflection/ClassReflection.php

```diff
@@ -465,6 +480,11 @@ public function getMethod(string $methodName, ClassMemberAccessAnswerer $scope):
 				}
 				$this->methods[$key] = $method;
 			}
+
+			if ($this->requireExtendsMethodsClassReflectionExtension->hasMethod($this, $methodName)) {
+				$method = $this->requireExtendsMethodsClassReflectionExtension->getMethod($this, $methodName);
+				$this->methods[$key] = $method;
```

Comment:
Why are you overriding a method that might have already been set in the foreach above?